tools/deployment/package-helm/templates/mcp-server-deployment.yaml

```
@@ -0,0 +1,96 @@
+            "type" "job"
+            "name" "results-cache-indices-creator"
+          ) | nindent 10 }}
+      containers:
+        - name: "mcp-server"
+          image: "{{ include "clp.image.ref" . }}"
+          imagePullPolicy: "{{ .Values.image.clpPackage.pullPolicy }}"
+          env:
+            - name: "CLP_DB_PASS"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "clp.fullname" . }}-database
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)